### PR TITLE
Adding the example program files that go along with the python-sqlite…

### PR DESCRIPTION
**Pull request by writeson**
…-sqlalchemy article

**Where to put new files:**

- New files should go into a top-level subfolder, named after the article slug. For example: `my-awesome-article`

**How to merge your changes:** 

1. [Make sure the CI code style tests all pass (+ run the automatic code formatter if necessary).](https://github.com/realpython/materials/blob/master/README.md)
2. Find an RP Team member on Slack and ask them to review & approve your PR.
3. Once the PR has one positive ("approved") review, GitHub lets you merge the PR.
4. 🎉

### CODE REVIEW

File: python-sqlite-sqlalchemy/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Pythoh / Sqlite /SqlAlchemy article
```
Comment (dbader) [inline python-sqlite-sqlalchemy/README.md]: Typo "pythoh"

File: python-sqlite-sqlalchemy/build_data/code/README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+ (temp_data.csv) in the data directory
+ containing temperature samples taken by students in a class.
+ The top row contains the labels, the students name followed
+ by a date value for each Wednesday of the week for a year.
+ 
+ It then creates data for each sample based on a table of 
+ temperature data, +/- 10 to make the data look variable
+ and reasonable.
+ 
+  ## Build the Temperature Database File
```
Comment (dbader) [inline python-sqlite-sqlalchemy/build_data/code/README.md]: There seems to be some extra-whitespace before the `##` headline indicator (also with other headlines below)

File: python-sqlite-sqlalchemy/build_data/code/README.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+ the data and the relationships between the data necessary
+ for the examples.
+ 
+ ## Directory Structure
+ 
+ The directory structure is set up in such a way the
+ build programs (as well as the examples) can find the
+ data files needed for each one.
+ 
+ # Executing the Programs
```
Comment (dbader) [inline python-sqlite-sqlalchemy/build_data/code/README.md]: Should this be a `##` (h2) headline to be consistent with the others above it?

File: understanding-asynchronous-programming/example_2.py [hunk 4]
```diff
@@ -1,4 +1,5 @@
 import queue
+from builtins import range
```
Comment (dbader) [inline understanding-asynchronous-programming/example_2.py]: Looks like this change is unrelated to the `python-sqlite-sqlalchemy` article—do we need it or is it accidental?

**Comment by dbader** [comment]
@writeson Had a couple of comments but otherwise your PR looks good & is ready to merge once these comments are resolved :) Happy New Year!

File: python-sqlite-sqlalchemy/build_data/code/README.md (hunk 3 above)
Comment (writeson) [reply python-sqlite-sqlalchemy/build_data/code/README.md]: Agreed

File: understanding-asynchronous-programming/example_2.py (hunk 4 above)
Comment (writeson) [reply understanding-asynchronous-programming/example_2.py]: Whoops, that was accidental.
